lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -172,20 +175,55 @@ def test_linkedin_share_url(self):
         test_url = get_certificate_url(course_id=self.course.id, uuid=self.cert.verify_uuid)
         response = self.client.get(test_url)
         self.assertEqual(response.status_code, 200)
-        self.assertIn(urllib.quote_plus(self.request.build_absolute_uri(test_url)), response.content)
+        params = OrderedDict([
+            ('_ed', '0_0dPSPyS070e0HsE9HNz_13_d11_',),
+            ('pfCertificationName', '{platform_name} Honor Code Certificate for {course_name}'.format(
+                platform_name=settings.PLATFORM_NAME,
+                course_name=self.course.display_name,
+            ),),
+            ('pfCertificationUrl', self.request.build_absolute_uri(test_url),),
+        ])
+        self.assertIn(
+            self.linknedin_url.format(params=urlencode(params)),
+            response.content
+        )
 
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
-    @mock.patch("microsite_configuration.microsite.is_request_in_microsite", _fake_is_request_in_microsite)
     def test_linkedin_share_microsites(self):
         """
-        Test: LinkedIn share URL should not be visible when called from within a microsite (for now)
+        Test: LinkedIn share URL should be visible when called from within a microsite.
         """
         self._add_course_certificates(count=1, signatory_count=1, is_active=True)
         test_url = get_certificate_url(course_id=self.cert.course_id, uuid=self.cert.verify_uuid)
-        response = self.client.get(test_url)
+        response = self.client.get(test_url, HTTP_HOST=settings.MICROSITE_TEST_HOSTNAME)
+        self.assertEqual(response.status_code, 200)
+        # the URL should be present
```

Comment:
It seems like we are validating more than just the presence of the URL parameter -- I would either add more context to this comment, assuming it is useful, or I would remove it.